docker/Dockerfile

```diff
@@ -44,14 +46,23 @@ RUN --mount=type=cache,target=/var/cache/sccache \
     just notice-file && \
     mv target/$(just arch=$TARGETARCH libc=gnu print-target)/release/restate-server target/restate-server
 
+FROM builder AS upload-false
+
+FROM builder AS upload-true
+RUN $(just --set arch $TARGETARCH --evaluate _arch)-linux-gnu-objcopy --only-keep-debug target/restate-server target/restate-server.debug
+RUN $(just --set arch $TARGETARCH --evaluate _arch)-linux-gnu-objcopy  --strip-debug target/restate-server
+RUN --mount=type=secret,id=parca parca-debuginfo upload --store-address=grpc.polarsignals.com:443 --bearer-token-file=/run/secrets/parca target/restate-server.debug
```

Comment:
Does that mean that we'll only keep a single/latest debuginfo file there or is there a way to attach this to the sha/version somehow?

Reply:
its indexed against the elf build id (which is also how debuginfod works)